src/main/java/MetersServlet.java

```diff
@@ -0,0 +1,44 @@
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/meters")
+public class MetersServlet extends HttpServlet {
```

Comment:
Tutaj dokładnie to samo co w `WeightServlet`

Reply:
nie rozumiem, czy chodzi o to że to jest dokładnie taki sam kod a inne parametry? nie wiem jak to zrobić lepiej

Reply:
nie wiem czy o to chodziło ale poprawione

Reply:
Chodziło o to że te samo błędy tutaj były :)